packages/flutter/test/foundation/binding_2_test.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+class TestBinding extends BindingBase { }
+
+void main() {
+  test('BindingBase.debugCheckZone', () async {
+    final BindingBase binding = TestBinding();
+    binding.debugCheckZone('test1');
+    Zone.current.fork().run(() {
+      expect(() { binding.debugCheckZone('test2'); }, throwsA(isA<FlutterError>()));
```

Comment:
assert that the thrown error actually contains some of the expected information to make sure we get the right one here?

Reply:
done (and hey, it was already the right one this time)